src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Tabs = ({ tabs, selectedTabId, tabChanged }) => (
+  <div>
+    <ul>
+      {tabs.map(tab => (
+        <li key={tab.id}>
+          <button
+            type="button"
+            onClick={() => tabChanged(tab)}
+          >
+            {tab.title}
+          </button>
+        </li>
+      ))}
+    </ul>
+    <div>
+      {tabs.find(tab => tab.id === selectedTabId).content}
+    </div>
```

Comment:
Враховуючи, що це текст, семантико-логічно його загорнути в \<p\>, наприклад
```suggestion
    <p>
      {tabs.find(tab => tab.id === selectedTabId).content}
    </p>
```